backend/src/test/java/gov/cdc/usds/simplereport/service/PersonServiceTest.java

```diff
@@ -100,7 +102,8 @@ void roundTrip() {
         null,
         false,
         false);
-    List<Person> all = _service.getAllPatients(PATIENT_PAGEOFFSET, PATIENT_PAGESIZE);
+    List<Person> all =
+        _service.getPatients(null, PATIENT_PAGEOFFSET, PATIENT_PAGESIZE, false, null);
```

Comment:
I considered it. The graphql api has to pass nulls, so I opted for the service to smell (stink?) like the function that calls it for consistency. Can this be refactored in a future changelist? (Also, would the search string parameter at the end also be `Optional<>`?)

The debate around passing Optional<> as parameter is a surprisingly contentious one? One point being you have to pass `@NotNull Optional<>` to make sure you don't have to test for 3 states instead of just two.
https://stackoverflow.com/questions/31922866/why-should-java-8s-optional-not-be-used-in-arguments#answers